cmd/eudico/daemon.go

```diff
@@ -135,6 +135,14 @@ func daemonCmd(consensusAlgorithm global.ConsensusAlgorithm) *cli.Command {
 				Name:  "restore-config",
 				Usage: "config file to use when restoring from backup",
 			},
+			&cli.BoolFlag{
+				Name:  "lite",
+				Usage: "start lotus in lite mode",
+			},
+			&cli.BoolFlag{
+				Name:  "mir-validator",
```

Comment:
Is this flag being used anywhere?

Reply:
When we run validators we need to start the daemon as `--mir-validator` so it only listens to mir and not to blocks propagated in the network (I was doing a dry-run of the spacenet deployment and that is how I realized these flags were missing).